web/src/app/shared/host-components-map/much-2-many/much-2-many.component.ts

```diff
@@ -38,6 +41,14 @@ export class Much2ManyComponent implements AfterViewChecked {
     return this.model.disabled;
   }
 
+  isHostDisabled() {
+    if (this.model?.mm === this.statusType.On) {
```

Comment:
hmm, may be? )))
```
return this.model?.mm === this.statusType.On
```

Reply:
Thanks, I forgot that condition was simplified